### PR TITLE
Delete unused Copyable component

### PR DESCRIPTION
**Pull request by whymarrh**
Refs #7506

This PR removes the unused `Copyable` component.

### CODE REVIEW

File: ui/app/components/app/copyable.js [hunk 1]
```diff
@@ -1,53 +0,0 @@
...
-}
-
-Copyable.prototype.render = function () {
-  const props = this.props
-  const state = this.state
-  const { value, children } = props
-  const { copied } = state
-
-  return h(Tooltip, {
-    title: copied ? this.context.t('copiedExclamation') : this.context.t('copy'),
```
Comment (Gudahtt) [inline ui/app/components/app/copyable.js]: Surprisingly, I think this is the only place the `copy` localized message is used. This should be deleted as well, assuming I'm not missing a case.

File: ui/app/pages/settings/index.scss [hunk 2]
```diff
@@ -205,10 +205,6 @@
     }
   }
 
-  &__copyable-address {
```
Comment (Gudahtt) [inline ui/app/pages/settings/index.scss]: :laughing: 
Funny that that class isn't even used by the `Copyable` component! I thought it was too though - I was planning to ask you to delete this if you'd forgotten.

Though it _is_ unused, so it should still be deleted.

Comment (whymarrh) [reply ui/app/pages/settings/index.scss]: It's unused and it has copyable in the name, how could I leave it 🔥

File: ui/app/components/app/copyable.js (hunk 1 above)
Comment (whymarrh) [reply ui/app/components/app/copyable.js]: I've double-checked and that does indeed look to be the case. Say no more, it's gone!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**